llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -10347,6 +10347,7 @@ SDValue RISCVTargetLowering::LowerINTRINSIC_WO_CHAIN(SDValue Op,
     return DAG.getNode(RISCVISD::MOPRR, DL, XLenVT, Op.getOperand(1),
```

Comment:
We need `setOperationAction(ISD::CLMUL, XLenVT, Legal)` in the constructor in this  file when `Subtarget.hasStdExtZbcOrZbkc()` is true.